Source/Scene/Model.js

```diff
@@ -2306,16 +2306,17 @@ define([
                     sampler : sampler,
                     flipY : false
                 });
-            }
-            // GLTF_SPEC: Support TEXTURE_CUBE_MAP.  https://github.com/KhronosGroup/glTF/issues/40
+                // GLTF_SPEC: Support TEXTURE_CUBE_MAP.  https://github.com/KhronosGroup/glTF/issues/40
 
-            if (mipmap) {
-                tx.generateMipmap();
+                if (mipmap) {
+                    tx.generateMipmap();
+                }
             }
         }
-
-        model._rendererResources.textures[gltfTexture.id] = tx;
-        model._texturesByteLength += tx.sizeInBytes;
+        if (defined(tx)) {
```

Comment:
This looks like another case that should never happen or a place we should have a Runtime or Developer error.

Reply:
@lilleyse ?

Reply:
Yeah this should be a Runtime error.